react/react-tests.ts

```diff
@@ -145,7 +146,7 @@ namespace StatelessComponent {
 
 var StatelessComponent2: React.SFC<SCProps> =
     // props is contextually typed
-    props => React.DOM.div(null, props.foo);
+    (props: SCProps) => React.DOM.div(null, props.foo);
```

Comment:
per the comment above this line, `props` is supposed to be contextually typed. So please leave out the `SCProps` annotation.